android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -1350,6 +1387,22 @@ public static boolean getMayPredictOverride() {
     return mayPredictOverride;
   }
 
+  /**
+   * Maps radio button index 0..3 to SuggestionType and then store as preference
+   * @param languageID as String
+   * @param suggestType Radio button index 0 to 3
+   */
+  public static void setMaySuggest(String languageID, int suggestType) {
+    if (suggestType < 0 || suggestType > 3) {
```

Comment:
Can we use `SuggestionType.SUGGESTIONS_DISABLED.toInt()` or something like that? Everywhere we have integer values in the java code?